packages/components/src/toggle-group-control/types.ts

```diff
@@ -17,6 +17,11 @@ export type ToggleGroupControlOptionProps = {
 	 * to specify a different label for assistive technologies.
 	 */
 	label: string;
+	/**
+	 * Icon for the option. If `icon` is provided it will be used instead of the `label`
+	 * and will show a tooltip automatically.
+	 */
+	icon?: JSX.Element;
```

Comment:
I'm not sure about the best type here... --cc @ciampo

Reply:
`JSX.Element` is the type I get back from the compiler for components in `icons/src/library/*`

<img width="325" alt="Screen Shot 2022-03-28 at 11 54 23 am" src="https://user-images.githubusercontent.com/6458278/160309708-40e7f1c2-244a-4d46-8cce-60395ab5796f.png">